.github/workflows/unit-test.yml

```
@@ -2,20 +2,138 @@ name: Unit Test
-    - uses: actions/checkout@v1
+    - uses: actions/checkout@v4
+    - name: Install Trivy
+      run: |
+        curl -sfL https://raw.githubusercontent.com/aquasecurity/trivy/main/contrib/install.sh | sh -s -- -b /usr/local/bin v0.48.1
     - name: Setup GitHub Action
       uses: actions/setup-node@v1
       with:
         node-version: 23
     - name: Install dependencies
       run: npm install
+        - name: Run Trivy Scan
```

actionlint (1.7.4)

could not parse as YAML: yaml: line 22: mapping values are not allowed in this context

(syntax-check)